vanuatuvoices/locale/eo/LC_MESSAGES/clld.po

```diff
@@ -230,5 +226,10 @@ msgstr ""
 
 #: vanuatuvoices/templates/dataset/detail_html.mako:25
 msgid "Cite the Vanuatu Voices dataset as"
-msgstr "Taem yu yusum o tekem wan infomeisen long ol Vois blong Vanuatu dataset, yu referensem olsem"
+msgstr ""
+"Taem yu yusum o tekem wan infomeisen long ol Vois blong Vanuatu dataset, "
+"yu referensem olsem"
```

Comment:
@Bibiko did you check that these wrapped translations work? I'm not sure the PO format allows this.